backend/server/test/unit-test/Makefile

```diff
@@ -1,7 +1,7 @@
 # Docker tests variables
 DOCKER_CONTAINER_BASE = manager_python_testing
 DOCKER_TESTED_BRANCH  = head
-DOCKER_REGISTRY       = registry.mgr.suse.de
+DOCKER_REGISTRY       = registry.opensuse.org
```

Comment:
I wonder if we should not just remove this file.

It says below that this is using an Oracle container?

Reply:
I think this is unused. AFAIK this is called from backend/test/docker-backend-server-tests.sh

Reply:
I could change `backend/test/docker-backend-server-tests.sh`, but then that means no tests for the backend anymore :-?

@meaksh?

Something tell me that I need to rollback this change for `backend/server/test/unit-test/Makefile` until we can clarify.

Reply:
I talked to Pablo. We are removing this file, and created https://github.com/SUSE/spacewalk/issues/15099